### PR TITLE
Feature/family finder

### PR DESCRIPTION
**Pull request by timmyg**
build: https://goo.gl/FGLHEG

### CODE REVIEW

File: front_end/src/app/admin/events/family-finder/household/household.component.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<header-event></header-event>
+
+<div class="container-fluid">
+  <preloader *ngIf="processing"></preloader>
+  <div class="mt-2 clearfix" *ngIf="eventParticipants && !processing">
```
Comment (jpc11623) [inline front_end/src/app/admin/events/family-finder/household/household.component.html]: Could this be consolidated into a function that can be tested in the component spec? We have two conditions performing some logic.

File: front_end/src/app/shared/components/child-signin-card/child-signin-card.component.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { Component, Input } from '@angular/core';
+
+import { Child, NewChild } from '../../models';
+
+@Component({
+  selector: 'child-signin-card',
+  templateUrl: 'child-signin-card.component.html',
+  // styleUrls: ['./_cards.scss', '../../../../assets/scss/_buttons.scss', ]
```
Comment (jpc11623) [inline front_end/src/app/shared/components/child-signin-card/child-signin-card.component.ts]: commented code

File: SignInCheckIn/MinistryPlatform.Translation/Repositories/ParticipantRepository.cs [hunk 3]
```diff
@@ -218,5 +218,29 @@ public List<MpGroupParticipantDto> GetGroupParticipantsByParticipantAndGroupId(i
...
+                "Household_ID_Table_Address_ID_Table.Address_Line_1",
+                "Household_ID_Table_Address_ID_Table.City",
+                "Household_ID_Table_Address_ID_Table.[State/Region] as State",
+                "Household_ID_Table_Address_ID_Table.Postal_Code",
+                "Household_ID_Table.Home_Phone",
+                "Contacts.Mobile_Phone",
+                "Household_ID_Table_Congregation_ID_Table.Congregation_Name"
+            };
+
+            var contacts = _ministryPlatformRestRepository.UsingAuthenticationToken(token).
```
Comment (dtkocher) [inline SignInCheckIn/MinistryPlatform.Translation/Repositories/ParticipantRepository.cs]: Should be searching on the display name.

File: front_end/src/app/admin/events/family-finder/household/household.component.html (hunk 1 above)
Comment (dtkocher) [inline front_end/src/app/admin/events/family-finder/household/household.component.html]: This stuff will be changing in the Story I am currently working so will update in that story.

**Comment by timmyg** [comment]
merging as dustin and john have reviewed, i just fixed a test

**Pull request merged**